setup/index.ts

```diff
@@ -54,6 +54,11 @@ function getInput(): Input {
   if (version.startsWith('v')) {
     throw new Error("'version' input should not be prefixed with 'v'");
   }
+  if (version.split('.').length !== 3) {
+    throw new Error(
+      "'version' input should specify the entire Teleport version e.g 11.0.1"
+    );
+  }
```

Comment:
I think we should do something like
 
```suggestion
  const versionRegex = /^([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$/i;
  
  if (!versionRegex.test(version)) {
    throw new Error(
      "incorrect 'version' specified, it should follow semantic versioning"
    );
  }
```

This regex works with all the tags here without the `v` prefix - https://github.com/gravitational/teleport/tags (`12.0.0-dev.justinas.8`, `11.0.2`, etc)